llvm/lib/SYCLLowerIR/ESIMD/LowerESIMD.cpp

```diff
@@ -600,7 +600,6 @@ class ESIMDIntrinDescTable {
         {"ustrunc_sat", {"ustrunc.sat", {a(0)}}},
         {"sutrunc_sat", {"sutrunc.sat", {a(0)}}},
         {"sstrunc_sat", {"sstrunc.sat", {a(0)}}},
-        {"abs", {"abs", {a(0)}, nk(-1)}},
```

Comment:
please don't remove, keep the lowering for cases like old static MKL library case.